onnxruntime/core/optimizer/matmul_transpose_fusion.cc

```diff
@@ -168,6 +168,15 @@ static Node* ReorderCastAndTranspose(Graph& graph, Node* cast,
   return &new_transpose;
 }
 
+// Check whether the element_type is an allowed FusedMatMul data type or not.
+static bool AllowedFusedMatMulDataType(ONNX_NAMESPACE::TensorProto_DataType element_type)
```

Comment:
Is it better to consult the op schema? Or is it safe to hardcode? I think the op scheman doesn't change that often.